Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SEC102_001.EmailAddress_with_email_addresses.sarif

```diff
@@ -39,7 +39,7 @@
       "artifacts": [
         {
           "location": {
-            "uri": "src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/Inputs/SEC102_001.EmailAddress_with_email_addresses.txt",
```

Comment:
>EmailAddress_with_email_addresses [](http://example.com/codeflow?start=105&length=33)

Why did someone think suffixing this file with 'with_email_addresses' provided more information? I don't know. Seems redundant to me so I trimmed it. :)